.github/workflows/e2e-browser.yml

```diff
@@ -7,90 +7,49 @@ env:
   CI: true
   DEBUG: "pw:webserver"
   ACTIONS_STEP_DEBUG: true
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 jobs:
-  build:
```

Comment:
@NSeydoux we actually should move towards this, as otherwise we are building N times when we only need to build once. I know it's inconsistent though across repos. Haven't had a chance to roll out "prebuilding the SDK" across to all repos — this is the whole "one pipeline" dream.

Reply:
Oh I see, I missed that. I've added it back with a short comment explaining this.